src/components/TodoFilter/TodoFilter.tsx

```
@@ -1,30 +1,57 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+import React from 'react';
+import { Select } from '../../types/Select';
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+interface TodoFilterProps {
+  search: string;
+  onChangeInput: (v: string) => void;
+  chooseStatus: (v: Select) => void;
+}
 
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button data-cy="clearSearchButton" type="button" className="delete" />
-      </span>
-    </p>
-  </form>
-);
+export const TodoFilter: React.FC<TodoFilterProps> = ({
+  search,
+  onChangeInput = () => {},
+  chooseStatus = () => {},
```

Issue: You are providing default values for `onChangeInput` and `chooseStatus` in the destructuring assignment. Since these props are required in the `TodoFilterProps` interface and always provided by the parent, default values are unnecessary and could hide missing prop issues. Remove the default values to ensure prop requirements are enforced.